packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -0,0 +1,304 @@
+                account_type: categoryAccountType,
+                address: settings?.address_line_1 ?? '',
+                city: settings?.address_city ?? '',
+                company: selectedJurisdiction,
+                country: settings?.country_code ?? '',
+                email: settings?.email ?? '',
+                leverage: availableMT5Accounts?.find(acc => acc.market_type === marketType)?.leverage ?? 500,
+                mainPassword: password,
+                ...(marketType === 'financial' && { mt5_account_type: 'financial' }),
+                ...(selectedJurisdiction &&
+                    (selectedJurisdiction !== 'labuan'
+                        ? {
+                              account_type: categoryAccountType,
+                              ...(selectedJurisdiction === 'financial' && { mt5_account_type: 'financial' }),
+                          }
+                        : {
+                              account_type: 'financial',
+                              mt5_account_type: 'financial_stp',
+                          })),
+                ...(marketType === 'all' && { sub_account_category: 'swap_free' }),
+                name: settings?.first_name ?? '',
+                phone: settings?.phone ?? '',
+                state: settings?.address_state ?? '',
+                zipCode: settings?.address_postcode ?? '',
+            },
+        });
+    }, [
+        ActiveTradingAccount?.is_virtual,
+        availableMT5Accounts,
+        isMT5PasswordNotSet,
+        marketType,
+        mutate,
+        password,
+        selectedJurisdiction,
+        settings?.address_city,
+        settings?.address_line_1,
+        settings?.address_postcode,
+        settings?.address_state,
+        settings?.country_code,
+        settings?.email,
+        settings?.first_name,
+        settings?.phone,
+        tradingPasswordChange,
+    ]);
+
+    const renderTitle = useCallback(() => {
+        if (isSuccess) {
+            return ' ';
+        }
+        if (hasMT5Account) {
+            return `Add a ${isDemo ? 'demo' : 'real'} ${PlatformDetails.mt5.title} account`;
+        }
+        return `Create a ${isDemo ? 'demo' : 'real'} ${PlatformDetails.mt5.title} account`;
+    }, [hasMT5Account, isDemo, isSuccess]);
+
+    const renderSuccessButton = useCallback(() => {
+        if (isDemo) {
+            return (
+                <div className='w-5000 sm:w-full'>
+                    <Button className='w-full' onClick={hide} size='lg'>
+                        OK
+                    </Button>
+                </div>
+            );
+        }
+        return (
+            <ButtonGroup className='justify-center w-full'>
+                <Button onClick={hide} size='lg' variant='secondary'>
+                    Maybe later
+                </Button>
+                <Button
+                    onClick={() => {
+                        hide();
+                        history.push('/wallets/cashier/transfer');
```

Comment:
Is this url is correct?
@thisyahlen-deriv @shontzu-deriv

Reply:
no pls change it

Reply:
should be `cashier/transfer`